main.nf

```diff
@@ -117,24 +114,23 @@ else{
   hairpin = file(params.hairpin)
 }
 
-// Validate inputs(NOT WORKING!!)
-// if( !mature.exists() ) exit 1, "Missing Bowtie 1 miRBase_mature index: ${mature}"
-// if( !hairpin.exists() ) exit 1, "Missing Bowtie 1 miRBase_hairpin index: ${hairpin}"
-// if( !index.exists() ) exit 1, "Missing Bowtie 2 index: ${index}"
-// if( !gtf.exists() )   exit 2, "Missing GTF annotation: ${gtf}"
-// if( !bed12.exists() ) exit 2, "Missing BED12 annotation: ${bed12}"
+// Validate inputs
+//if( !mature.exists() ) exit 1, "Missing Bowtie 1 miRBase_mature index: ${mature}"
+//if( !hairpin.exists() ) exit 1, "Missing Bowtie 1 miRBase_hairpin index: ${hairpin}"
+//if( !index.exists() ) exit 1, "Missing Bowtie 2 index: ${index}"
+//if( !gtf.exists() )   exit 2, "Missing GTF annotation: ${gtf}"
+//if( !bed12.exists() ) exit 2, "Missing BED12 annotation: ${bed12}"
```

Comment:
Please note that for mature miRNA and hairpin, index, gtf and bed12 are not used.